packages/backend/src/nest/registration/registration.validators.ts

```diff
@@ -3,6 +3,7 @@ import { registerDecorator, ValidationArguments, ValidationOptions } from 'class
 import Logger from '../common/logger'
 
 const logger = Logger('registration.validators')
+
 export function IsCsr(validationOptions?: ValidationOptions) {
```

Comment:
Do I get it right, that we want to introduce the same method for certificate? Also, what are we particularly interested in validating inside `CertificateContainsFields`?